src/services/emailService.js

```diff
@@ -0,0 +1,56 @@
+const nodemailer = require('nodemailer');
+
+require('dotenv').config();
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASSWORD,
+  },
+});
+
+const send = ({ email, subject, html }) => {
+  return transporter.sendMail({
+    to: email,
+    subject,
+    html,
+  });
+};
```

Comment:
Consider adding error handling for the `send` function to catch and log any errors that occur during the email sending process. This will help in diagnosing issues if emails fail to send.